src/client/Microsoft.Identity.Client/json/Bson/BsonBinaryType.cs

```diff
@@ -25,6 +25,8 @@
 
 using System;
 
+#nullable disable
```

Comment:
Try this:
In the .editorconfig, specify generated_code = true in a section that applies to that file.

This should disable nullable stuff (and a lot of other warnings)
See: https://docs.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies

> 
> Important
> 
> The global nullable context does not apply for generated code files. Under either strategy, the nullable context is disabled for any source file marked as generated. This means any APIs in generated files are not annotated. There are four ways a file is marked as generated:
> 
>     In the .editorconfig, specify generated_code = true in a section that applies to that file.
>     Put <auto-generated> or <auto-generated/> in a comment at the top of the file. It can be on any line in that comment, but the comment block must be the first element in the file.
>     Start the file name with TemporaryGeneratedFile_
>     End the file name with .designer.cs, .generated.cs, .g.cs, or .g.i.cs.
> 
> Generators can opt-in using the [#nullable](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#nullable-context) preprocessor directive.